src/utils/websocket.js

```diff
@@ -19,11 +19,19 @@ export function constructWebsocketURL(type, period) {
   return `${proto}//${loc.host}${prefix}${prefix.endsWith('/') ? '' : '/'}v1/ws/${period}/${type}`
 }
 
-export function wsChanges(dispatch, type, period) {
+export function wsChanges(dispatch, type, period, ns) {
```

Comment:
Do you need to return ws object at the end of function? (The same questions menteiond in backingImage.js)

Reply:
I called updateWs under different modals internally (depending on `ns`, it can be dispatched to different modals). Assign the created ws to different modal state ws.